src/main/java/com/service/AuthorisationService.java

```diff
@@ -0,0 +1,33 @@
+package com.service;
+
+import com.dao.UserDAO;
+import lombok.Data;
+
+import javax.sql.DataSource;
+
+@Data
+public class AuthorisationService {
+
+    private DataSource dataSource;
```

Comment:
What's the purpose of this field? You don't use it

Reply:
removed